Update RUM Next.js guide to recommend CDN method for Next.js app router (#21226)
DataDog/documentation: content/en/real_user_monitoring/guide/monitor-your-nextjs-app-with-rum.md

1. Choose your instrumentation type, then copy and paste the code snippet from the Datadog RUM UI into the appropriate file based on the instrumentation type.

   {{< tabs >}}
   {{% tab "npm" %}}

>    If using npm, you need to make a few small changes to the code snippet from the Datadog RUM UI before pasting it into either the root `layout.tsx` or custom `_app.tsx` file (Datadog supports both):

Reviewer: ```suggestion
   When using npm, you need to make a few small changes to the code snippet from the Datadog RUM UI before pasting it into either the root `layout.tsx` or custom `_app.tsx` file (Datadog supports both):
```

>    - If your application relies on the **newer** Next.js [App Router][1] (versions 13+), we recommend using the CDN RUM instrumentation to ensure that the RUM initialization occurs in the client. If you would prefer to use the npm package, the initialization code must run in a [client component][5] so that RUM can collect telemetry from the client. You can achieve this without making your [`layout.tsx`][2] file itself a client component by following the example below to create an empty `<DatadogInit />` component with the RUM initialization code, and then rendering that `<DatadogInit />` component in your `layout.tsx`.

Reviewer: ```suggestion
   - If your application relies on the **newer** Next.js [App Router][1] (versions 13+), Datadog recommends using the CDN RUM instrumentation to ensure that the RUM initialization occurs in the client. If you want to use the npm package, the initialization code must run in a [client component][5] so RUM can collect telemetry from the client. You can achieve this without making your [`layout.tsx`][2] file itself a client component by following the example below to create an empty `<DatadogInit />` component with the RUM initialization code, and then rendering that `<DatadogInit />` component in your `layout.tsx`.
```
